x-pack/plugins/ml/public/services/ml_api_service/index.js

```diff
@@ -27,6 +27,12 @@ export const ml = {
     });
   },
 
+  getMultipleJobs(jobIds) {
```

Comment:
Looks like this new function might not be needed if https://github.com/elastic/kibana/pull/31763/files#r260769619 can be implemented.

Reply:
Yeah - removed in a861b7d09a146889329f732a1f2fd651561a5086